docs/DeveloperGuide.adoc

```diff
@@ -727,11 +727,48 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
+|`* * *` |user |delete multiple persons at once  |avoid going through the hassle of removing one at each time
+
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |select multiple person |perform the same command to multiple contacts at once
+
+|`* * *` |user |update my contact |I do not have to recreate a contact whenever there are changes
+
+|`* * *` |user |store multiple information of the same kind for a contact |accommodate contacts with multiple addresses, e-mail addresses or phone numbers
+
+|`* * *` |user |get suggestions on the tags I have used |be consistent with the tags I use
+
+|`* *` |user |see a sorted list of my contacts |it will save time and energy on my end
+
+|`* *` |user |use shortcut to any command |it will be easier for me to find someone in my contact list
+
+|`* *` |user |see a summarised list of shortcut commands |refer to the corresponding shortcut for the command I want to use
+
+|`* *` |user |rename existing tag |update my relationship with contact
+
+|`* *` |user |link my contact's saved email to a 'compose email page' |send an email to him/her with a click
```

Comment:
Take note that one of the project constraint is that the primary input mode should be CLI.